packages/core/src/App/Containers/RealAccountSignup/account-wizard.jsx

```diff
@@ -332,6 +332,12 @@ const AccountWizard = observer(props => {
         // Check if account wizard is not finished
         if (should_override || index + 1 >= state_items.length) {
             createRealAccount({});
+            //to count the last step of the wizard 'terms_of_use' as a step
+            trackEvent({
+                action: 'step_passed',
+                step_num: 3,
+                step_codename: STEP_IDENTIFIERS[3],
```

Comment:
is it possible to use `current_step` variable here too? just like line `271`

Reply:
asking this because using hardcoded strings/keys can cause issues in the future, imagine that we add another step here! then we may need to change this manually.

Reply:
@ali-hosseini-deriv Yes you are right I did it intentionally to check on the data we get it's an experiment and we may need to go with more improvement in different iteration, you can check the card. If you still think we shouldn't go with hard coded index let me know and I will change it , thanks 🙏

Reply:
why its needed to be hardcoded? is there something which is blocking you?
if yes, you can go ahead with the current one, if not I suggest to do it in one shot with the best possible implementation.

Reply:
No there is nothing blocking me I just wanted to be accurate on the step to see how it affects the data, but sure I will go by your suggestion. Let me just test it with consoling it and I will apply the changes thanks. 🙇‍♀️